### DISCUSSION TITLE
Problems compiling / linking master branch

### DISCUSSION THREAD

**Discussion by MelleNooijen** [vgmtrans/vgmtrans#234] [Q&A]
I've been trying to build the `master` branch after the recent updates. However, first time I've tried this, when building, my system ran out of memory (8 GB physical + 8 GB swap). Trying again (after allocating more swap space) I get the following error(s):
```
make[2]: *** [CMakeFiles/Makefile2:207: src/main/CMakeFiles/vgmtranscore.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:243: src/ui/qt/CMakeFiles/vgmtrans.dir/rule] Error 2
make: *** [Makefile:215: vgmtrans] Error 2
```
Am I doing something wrong, or is this regular behaviour and is the `master` branch just (still) broken?

_(As for system info, I am running the latest versions of software available in the Manjaro Stable repos... sorry for not being more specific, I am new to Manjaro, coming from Ubuntu :sweat_smile: )_

**Comment by sykhro** [comment]
`master` is not broken, actually, it's gonna see a release soon. I regularly build it on a 8gb machine.
The log you posted says nothing about the actual error, please post a complete one.

**Reply by MelleNooijen** [reply]
There were no further log details above, just normal `cmake` output (`Built target [...]`). But the issue seems to have fixed itself after a third attempt, so sorry for being too fast in posting and possibly wasting your time.

**Reply by sykhro** [reply]
No worries, I'm very interested in build failures so lmk if you manage to reproduce
